module/src/components/switchInput/switchInput.theme.scss

```diff
@@ -1,9 +1,6 @@
-@use "../../theme/variables.scss";
-@use "../../theme/mixins.scss";
-
 $arm-switch-input-width: 50px !default;
 $arm-switch-input-height: 30px !default;
-$arm-switch-input-handle-padding: variables.$spacing-xxsmall !default;
+$arm-switch-input-handle-padding: var(--arm-spacing-xxsmall);
 $arm-switch-input-handle: calc(
```

Comment:
SCSS driven CSS variables used for all variables

Sensible level of abstraction of values to variables

No fixed pixel heights

No important tags or nested selectors